.github/workflows/test_build.yml

```diff
@@ -209,7 +209,7 @@ jobs:
               make validate-html
             else
             # otherwise, just validate HTML docs for packages that have been updated
-              PACKAGES=$(git diff --stat origin/development HEAD -- ../../Macaulay2/packages/ | grep -Po "(?<=Macaulay2/packages/)[^/\.]*(?=\.m2|/)" | uniq | xargs)
```

Comment:
What is this doing?  Is it now going to try to install these packages?

Reply:
No -- for every pull request, we do a `git diff` to see if any packages were updated, and if so, validate the html of these packages' docs.  In particular, we look for any changes to an .m2 file of `M2/Macaulay2/packages` or something in a subdirectory of `M2/Macaulay2/packages` to generate a list of packages to check.

However, `undistributed-packages` is a subdirectory of `M2/Macaulay2/packages` but it isn't a package!  So we were trying to validate the html docs of a nonexistent package, leading to occasional failing GitHub builds.  So now we filter this case out.

Reply:
Ah, thanks, that sounds good!